extra/onnx.py

```diff
@@ -159,7 +159,7 @@ def fetch_tensor(x: str):
         ret = inp[0].reshape([s for i,s in enumerate(inp[0].shape) if i not in axes])
       elif n.op_type == "Div":
         # in openpilot, due to SHUFFLE_PAD_OPS issues, we are spending an extra kernel
-        ret = inp[0].div(inp[1]) if inp[0].dtype == dtypes.float else inp[0].div(inp[1]).floor()
+        ret = inp[0].div(inp[1])
```

Comment:
The kernel for .floor in master is wrong since the values don't truncate to int32:
```
__kernel void E_4_32_4(__global float* data0, const __global float* data1, const __global float* data2, const __global float* data3) {
  int gidx0 = get_group_id(0); /* 4 */
  int lidx1 = get_local_id(0); /* 32 */
  float4 val0 = (float4)(*((__global float4*)(data1+(gidx0*128)+(lidx1*4))));
  float4 val1 = (float4)(*((__global float4*)(data2+(gidx0*128)+(lidx1*4))));
  float val2 = data3[0];
  float alu0 = (((val0).x+(val1).x)/val2);
  float alu1 = (((val0).y+(val1).y)/val2);
  float alu2 = (((val0).z+(val1).z)/val2);
  float alu3 = (((val0).w+(val1).w)/val2);
  *((__global float4*)(data0+(gidx0*128)+(lidx1*4))) = (float4)(float4)(((alu0<alu0)!=0?(alu0-1.0f):alu0),((alu1<alu1)!=0?(alu1-1.0f):alu1),((alu2<alu2)!=0?(alu2-1.0f):alu2),((alu3<alu3)!=0?(alu3-1.0f):alu3));
}
```

the code in this branch would be:
```
__kernel void E_4_32_4(__global float* data0, const __global float* data1, const __global float* data2, const __global float* data3) {
  int gidx0 = get_group_id(0); /* 4 */
  int lidx1 = get_local_id(0); /* 32 */
  float4 val0 = (float4)(*((__global float4*)(data1+(gidx0*128)+(lidx1*4))));
  float4 val1 = (float4)(*((__global float4*)(data2+(gidx0*128)+(lidx1*4))));
  float val2 = data3[0];
  float alu0 = (((val0).x+(val1).x)/val2);
  float alu1 = (((val0).y+(val1).y)/val2);
  float alu2 = (((val0).z+(val1).z)/val2);
  float alu3 = (((val0).w+(val1).w)/val2);
  int cast0 = (int)(alu0);
  int cast1 = (int)(alu1);
  int cast2 = (int)(alu2);
  int cast3 = (int)(alu3);
  *((__global float4*)(data0+(gidx0*128)+(lidx1*4))) = (float4)(float4)((float)(((alu0<cast0)!=0?(cast0-1.0f):cast0)),(float)(((alu1<cast1)!=0?(cast1-1.0f):cast1)),(float)(((alu2<cast2)!=0?(cast2-1.0f):cast2)),(float)(((alu3<cast3)!=0?(cast3-1.0f):cast3)));
}
```

applying floor with its truncate part will give the wrong result:
```
Not equal to tolerance rtol=0.01, atol=0.0001

Mismatched elements: 5269 / 6120 (86.1%)
Max absolute difference: 147.69875
Max relative difference: 105.37706
 x: array([[ 2.669612, -0.469016,  0.007674, ...,  0.      ,  0.      ,
         0.      ]], dtype=float32)
 y: array([[ 3.16809 , -0.461886,  0.010212, ...,  0.      ,  0.      ,
         0.      ]], dtype=float32)
```
that's why I'm removing it

Reply:
If you remove the .floor() from what's already in master, the kernel counts will end up being 210 - This PR didn't impact the kernel counts directly.